### PR TITLE
Start elasticsearch and verify it does what it should after installing in custom dir

### PR DESCRIPTION
**Pull request by brwe**
…stalling in custom dir

### CODE REVIEW

File: qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash [hunk 1]
```diff
@@ -93,6 +93,12 @@ fi
     chown -R elasticsearch:elasticsearch "$ESPLUGINS"
 
     install_jvm_example
+    start_elasticsearch_service
```
Comment (nik9000) [inline qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash]: I wonder if you can get away with doing this particular check while elasticsearch is started with all the plugins installed? Its fine if we can't though.

File: qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash [hunk 2]
```diff
@@ -93,6 +93,12 @@ fi
...
+    # check that configuration was actually picked up    
+    curl -s localhost:9200/_cat/configured_example | sed 's/ *$//' > /tmp/installed
+    echo "foo" > /tmp/expected
+    diff /tmp/installed /tmp/expected
```
Comment (nik9000) [inline qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash]: This returns non-0 if these are different? I'm pretty sure it does but I just want to double check.

Comment (brwe) [inline qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash]: yes it does. I tried it out too.

File: qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash (hunk 1 above)
Comment (brwe) [inline qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash]: the idea was to install in a custom folder and then check if elasticsearch starts and the plugin is working. do you mean we should install all plugins in custom folder?

Comment (nik9000) [inline qa/vagrant/src/test/resources/packaging/scripts/plugin_test_cases.bash]: Got it! You've got this right I think then.

**Comment by nik9000** [comment]
LGTM

**Pull request merged**